### DISCUSSION TITLE
如何同时标绘2种折线，一个闭合一个不闭合，并且可以修改属性不互相影响

### DISCUSSION THREAD

**Discussion by webdq** [zouyaoji/vue-cesium#206] [Q&A] [answered]
如何同时标绘2种折线，一个闭合一个不闭合，并且可以修改属性不互相影响

**Comment by zouyaoji** [comment, marked as answer]
画的时候动态设置，然后画完来改哈。
打开在线文档打开，画2个线，打开浏览器调试工具，可测试。
`vm.$refs.drawingsRef.getDrawingActionInstances()[2].cmpRef.value.renderDatas.value[1].loop = false`

整体设计思路是先取renderDatas中的配置（可以每个线不同），如果没有就取props中的配置（每个线相同）。
